source/sass/views/shared/sticky-cta.scss

```diff
@@ -3,11 +3,13 @@
 }
 
 .cta {
+  $sticky-cta-z-index: $zindex-sticky;
```

Comment:
it feels like we need a comment that points to https://getbootstrap.com/docs/4.0/layout/overview/#z-index here, even if bootstrap offers a built-in var - zindices are tricky as heck, so explaining that we need something that's lower than our header (which uses 1030 because it's sticky-fixed) is a good idea.

Reply:
Done. Added notes & reference pointing to https://v4-alpha.getbootstrap.com/layout/overview/#z-index